app/account/page.jsx

```diff
@@ -0,0 +1,109 @@
+'use client'
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import Tabs from '@mui/material/Tabs';
+import Tab from '@mui/material/Tab';
+import Typography from '@mui/material/Typography';
+import Box from '@mui/material/Box';
+import { useEffect, useState } from 'react'
+import { useUserContext } from '../context/usercontext'
+import { useRouter } from 'next/navigation'
+import Profile from '../components/profile'
+
+function CustomTabPanel(props) {
+  const { children, value, index, ...other } = props;
+
+  
+
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`simple-tabpanel-${index}`}
+      aria-labelledby={`simple-tab-${index}`}
+      {...other}
+    >
+      {value === index && (
+        <Box sx={{ p: 3 }}>
+          <Typography>{children}</Typography>
+        </Box>
+      )}
+    </div>
+  );
+}
+
+CustomTabPanel.propTypes = {
```

Comment:
personal preference here but I typically like to have my propTypes listed at the top of the file right before the component is defined. This way I can quickly see what props are being passed into the component rather than needing to scroll down to view this info.